### PR TITLE
WinUI 884790 prepare the UG document for bar chart in SfCartesian chart control

### PR DESCRIPTION
**Pull request by Harsha-SF4223**
Prepared the Bar chart UG document content for SfCartesian chart control in WinUI
I have added below sections in the document for bar chart
1)Bar chart
2)SegmentSpacing

### CODE REVIEW

File: winui/Cartesian-Charts/Bar.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+description: Learn here all about bar chart support in Syncfusion WinUI Chart (SfCartesianChart) control.
+platform: WinUI
+control: SfCartesianChart
+documentation: ug
+---
+
+# Bar Chart in WinUI Chart (SfCartesianChart)
+
+A bar chart uses bars to represent data points and compare values across different categories. To render bar chart, initialize the [SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html) and switch the chart X and Y axes by using the [IsTransposed](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_IsTransposedProperty) property as true. Then, create the column chart. To render a column chart, create an instance of [ColumnSeries](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ColumnSeries.html), and add it to the [Series](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html#Syncfusion_UI_Xaml_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html).
+
```
Comment (NanthiniMahalingam) [inline winui/Cartesian-Charts/Bar.md:13]: @Harsha-SF4223 , change the bar chart description based on suggestion

File: winui/Cartesian-Charts/Bar.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    chart.Series.Add(series);
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Bar segment spacing in WinUI Chart](Chart-Types_images/WinUI_bar_chart_segment_spacing.png)
+
```
Comment (NanthiniMahalingam) [inline winui/Cartesian-Charts/Bar.md:119]: Add the note section for bar chart and ensure other chart types demo link.

File: winui/Cartesian-Charts/Bar.md (hunk 1 above)
Comment (Harsha-SF4223) [reply winui/Cartesian-Charts/Bar.md:13]: I have changed the description.

File: winui/Cartesian-Charts/Bar.md (hunk 2 above)
Comment (Harsha-SF4223) [reply winui/Cartesian-Charts/Bar.md:119]: Added note for the bar chart demo link and changed the demo links for other chart types.

**Review by SaiyathAliFathima — APPROVED** [review]

**Review by Saravanan-Madhesh — APPROVED** [review]
Fine

**Pull request merged**